### ISSUE TITLE
Selecting device Area with mouse on Windows MS Edge/Chrome ignored. Keyboard selection fine.

### ISSUE THREAD

**Issue by wigster** [home-assistant/frontend#8599]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Using MS Edge v89 (based off Chromium) on Windows 10. Firefox is ok.

When I go to a device and attempt to pick a new area for it, clicking on an option from the dropdown is ignored. Selecting it with a keyboard and confirming with Enter works ok.

This seems to be a generic issue with dropdown lists (e.g. entity picker in Entities card UI.).

### Describe the behavior you expected

Click with mouse, change area. 

### Steps to reproduce the issue

1. Open Device properties panel
2. Click on Area dropdown
3. Select new Area
4. Area not updated.
...

### What version of Home Assistant Core has the issue?

core-2021.3.3

### What was the last working version of Home Assistant Core?

?

### In which browser are you experiencing the issue with?

Edge v89 and v90

### Which operating system are you using to run this browser?

Windows 10

### State of relevant entities

```yaml
# Paste your state here.

```

### Problem-relevant frontend configuration

```yaml
# Paste your YAML here.

```

### Javascript errors shown in your browser console/inspector

```txt
# Paste your logs here.

```

**Comment by ngseer** [comment]
I can confirm this bug is presented for quite a while.
Entity selector in visual editor for some cards (e.g. Entities Card) also has been broken at the same time.

**Comment by ngseer** [comment]
Still can reproduce it (Home Assistant 2021.6.6., Chrome 91.0.4472.114).
Just let me know if there's any other relevant info I can provide to help with the issue.

**Comment by spacegaier** [comment]
I cannot reproduce the issue with Edge 94. Selecting of an area with the mouse works, in fact I cannot use the keyboard here, but I think you never could in the past.

I am not sure when the new device page / layout was introduced, so perhaps this issue applied to the old one where the area selection was not in the combined edit dialog with the device name?

**Comment by wigster** [comment]
In fact this seems to have been fixed at some point around 2021.7/8. In any case in .9 the issue is not there anymore.
